index.html

```diff
@@ -0,0 +1,160 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>GGP Restaurant Food and Drink</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="nav">
+          <div class="nav_container">
+            <img src="./images/facebook.svg" alt="facebook-logo" class="icon"/>
+            <img src="./images/twitter.svg" alt="twitter-logo" class="icon"/>
+            <ul class="nav_menu">
+              <li class="nav_menu-item"><a href="#">English</a></li>
+              <li class="nav_menu-item"><a href="#">My Page</a></li>
+              <li class="nav_menu-item"><a href="#">Logout</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="bottom-nav">
+          <div class="bottom-nav_center hidden" id="menu">
+            <a class="logo" href="index.html">
+                <image src="./images/logo.png" alt="logo" class="icon-image"/>
+            </a>
+
+            <ul class="nav-list">
+              <li class="nav-list_item"><a href="index.html">Home</a></li>
+              <li class="nav-list_item"><a href="about.html">About Us</a></li>
+              <li class="nav-list_item"><a href="#chefs">Our Chef</a></li>
+              <li class="nav-list_item"><a href="#menus">Menus</a></li>
+              <li class="nav-list_item"><a href="#testimonials">Testimonials</a></li>
+              <li class="nav-list_item"><a href="#partners">Partners</a></li>
+              <li class="nav-list_item ggp-restaurant">
+                <a href="#index.html">GGP Food&Drink</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="bottom-nav_hamburger" id="hambuger">
+          <img src="./images/hamburger.svg" alt="hamburger" class="ham-icon" id="openHamburger"/>
+          <img src="./images/closeicon.svg" alt="hamburger" class="hidden ham-icon" id="closeHamburger"/>
+        </div>
+      </nav>
+      <div class="restaurant" id="about">
+        <h3 class="restaurant_title">"A haven for food lovers, where every dish tells a unique story."</h3>
+        <h1 class="restaurant_heading">
+          Gyi Gyi Paw<br />
+          Food and Drink <br />
+          Restaurant
+        </h1>
+        <p class="restaurant_body">
+            Ask if the customers would like any assistance finding a table or if they have any specific seating
+            preferences. Offering assistance shows that our value their comfort and are ready to cater to their needs.
+        </p>
+        <p class="restaurant_date">Daily Open(9:00AM ~ 1:00PM)</p>
+        <p class="restaurant-email">gyigyipaw@gmail.com</p>
+      </div>
+    </header>
```

Comment:
- [ ] I see that you put the landing page content in the header tag,but it will be better if you put it in a separate section in the main tag for the semantics and to help the SEO👌😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Landing page| 
 | --- |
 |    ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/c670fa74-95b2-4f2f-afac-ed46d1a652fb)|